freqtrade/freqtradebot.py

```diff
@@ -433,7 +433,7 @@ def get_trade_stake_amount(self, pair: str) -> float:
                 Trade.total_open_trades_stakes()
             )
         else:
-            stake_amount = self.config['stake_amount']
+            stake_amount = self.strategy.get_stake_amount(pair)
```

Comment:
Please use `strategy_safe_wrapper()` for this - i assume it'll be user provided code - so should assume that it can cause errors.